integration_testing/features/super-admin/super-admin-windows-installer-setup-error-log.feature

```diff
@@ -0,0 +1,23 @@
+Feature: Super admin can see error logs during a failed Windows installer setup
+    Super admin needs to be able to see the error logs when the Kolibri Windows installation failed
+
+    Background:
+        Given that I have the Kolibri Windows installer
+          And I am using a Windows environment
+          And I double click the Kolibri Windows installer
+
+    Scenario: Install the kolibri Windows installer with the error
```

Comment:
This `Scenario` still has no `Given` or `When` statements after it.  Is this intentional?